src/Smp.Web/ClientApp/src/app/profile/profile.component.ts

```diff
@@ -17,14 +17,15 @@ import { RelationshipType } from '../models/relationship-type.enum';
   styleUrls: ['./profile.component.scss']
 })
 export class ProfileComponent implements OnInit {
-  @ViewChild(FeedComponent, { static: false }) feedComponent: FeedComponent;
+  @ViewChild(FeedComponent)
```

Comment:
the object with static in it is no longer needed as it defaults to static in Angular 9.